lib/screens/group_member_list/widgets/create_list_tile.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+///This class returns a list tile which would hold the name and details of member
+///It receives a string as parameter which is the name of the person
+
+class ListTileCreate extends StatelessWidget {
+  final String title;
+
+  ListTileCreate(this.title);
```

Comment:
1. Please take in profilePicUrl(optional) and memberId(compulsory) also. 
2. If the profilePicUrl is null then show account_circle else pic from that URL
3. Use named arguments :innocent: